source/material_model/simpler.cc

```diff
@@ -65,6 +73,15 @@ namespace aspect
           out.specific_heat[i] = reference_specific_heat;
           out.thermal_conductivities[i] = k_value;
           out.compressibilities[i] = 0.0;
+
+          derivatives->dviscosities_du[i] = 0.0;
```

Comment:
I am not sure what you mean, because the derivative is zero in this case. Should I just not set it and leave it at the default value of signaling:nan?